comment-updater-headless/src/main/kotlin/HeadlessConfig.kt

```diff
@@ -0,0 +1,4 @@
+object HeadlessConfig {
+    val OUTPUT_DIR_PATH = "/Users/Ivan.Pavlov/IdeaProjects/CommentUpdater/dataset"
```

Comment:
please, think about making it configurable via command line interface

Reply:
Done!